pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -19,7 +19,7 @@
       - name: Core services
         description: Suite of core Kubernetes applications every EKS cluster should have, including AWS LoadBalancer controller, external-dns, cluster-autoscaler, and more.
         plans: ["Standard", "Enterprise"]
-      - name: CI/CD
+      - name: Basic CI/CD
```

Comment:
I switched this back based on https://github.com/gruntwork-io/gruntwork-io.github.io/pull/579#discussion_r686719064, but to explain my original reasoning for changing this:

I felt weird calling this "Basic CI/CD" because in some sense this is more advanced as it includes a full workflow engine. The k8s-native solutions don't include workflow engines, so you can't do things like "run test on code and then deploy" - it goes straight to deploy.

What do you think about changing this to "Traditional CI/CD" and "K8s native CI/CD"?

Reply:
Does Argo CD not work with, say, [Argo Workflows](https://argoproj.github.io/argo-workflows/)?

Reply:
They work, but their really two very different systems. I'm not quite sure I follow what you are trying to get at here.

Reply:
Standard / Enterprise seems fine to me, so let's just keep it that way.

Reply:
> They work, but they're really two very different systems. I'm not quite sure I follow what you are trying to get at here.

You wrote:

> The k8s-native solutions don't include workflow engines, so you can't do things like "run test on code and then deploy" - it goes straight to deploy.

So I was wondering if you could implement a similar workflow using Argo Workflows, and trigger it on commits / PRs / etc using Argo CD. In other words, is what you were calling "Traditional CI / CD" really "more advanced" because it supports a "full workflow engine" or can the same things be done with Argo Workflows and therefore, K8S-native CI / CD? Not critical to resolve now, more of a curiosity thing.

Reply:
Ah gotcha. I was really going for this

> In other words, is what you were calling "Traditional CI / CD" really "more advanced" because it supports a "full workflow engine"

Argo Workflows does indeed give you the same things on Kubernetes, but so does Jenkins-X, or Jenkins with Kubernetes Runners plugins.

What I really wanted to differentiate was between a first class Kubernetes CI/CD experience with self-service and deployment dashboards (which is what Argo CD and Flux gives you), and one that only gives you progress information at the pipeline level, including having to define your custom pipeline in whatever format the pipeline engine requires.

I didn't know what was the right term to use for that, but basic vs advanced felt wrong. Standard vs Enterprise feels a bit better.

Reply:
Roger!